Fix order of maven repositories.

Adapted from [1], via [2].

[1]: https://github.com/johnsonmh/flutter/commit/cea908a939c520eff59173fcf42967a5869a8bbe
[2]: https://github.com/flutter/flutter/pull/23397#issuecomment-433069311

Apply this under flutter/ with -p1.

---
diff --git a/packages/flutter_tools/gradle/flutter.gradle b/packages/flutter_tools/gradle/flutter.gradle
index 30f262c58b4..e292b186095 100644
--- a/packages/flutter_tools/gradle/flutter.gradle
+++ b/packages/flutter_tools/gradle/flutter.gradle
@@ -18,10 +18,8 @@ import org.gradle.api.tasks.bundling.Jar
 
 buildscript {
     repositories {
+        google()
         jcenter()
-        maven {
-            url 'https://dl.google.com/dl/android/maven2'
-        }
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:3.2.1'
